### DISCUSSION TITLE
Recuperare pulsanti custom

### DISCUSSION THREAD

**Discussion by Pico1965** [indomus/forum#1392] [HUB: Home Assistant]
Buonasera.

Ho creato alcuni pulsanti custom con button card.

Ho creato il full backup

Ho reinstallato HA e volevo recuperare il codice dei pulsanti creati.

Ho pensato di reinstallare il backup ma non ha funzionato perchè era attivo il 2FA e l'applicazione genera codici per l'installazione successiva a quella che è di fatto inaccessibile.

Dal backup salvato dove trovo il codice dei pulsanti?

**Comment by snakuzzo** [comment]
quella roba viene salvata sotto la subdir `.storage` e credo in uno dei file `lovelace`

**Comment by Pico1965** [comment]
Grazie dell'info.

Aprendo il .tar del backup
homeassistant.tar.gz --> homeassistant.tar --> .storage --> lovelace (senza estensione)

**Reply by snakuzzo** [reply]
Ti chiediamo la cortesia di fare reply sotto l'ultimo messaggio e di non aprire nuovi thread (come fatto ora)
